concurrency/logging.properties

```diff
@@ -0,0 +1,40 @@
+# Ensure that both your client and sever JVMs point to this file using the java.util.logging property
+# -Djava.util.logging.config.file=/path/to/logging.properties
+
+#Handlers we plan to use
+handlers=java.util.logging.FileHandler,java.util.logging.ConsoleHandler
+
+#Global logger - By default only log warnings
+.level=WARNING
```

Comment:
Is there a reason we only want warnings?

Reply:
that was the config coming from the "starter" tck runner project from Jakarta... changed to INFO